src/main/java/io/confluent/connect/hdfs/wal/FSWAL.java

```diff
@@ -60,12 +61,17 @@ public void append(String tempFile, String committedFile) throws ConnectExceptio
       writer.append(key, value);
       writer.hsync();
     } catch (IOException e) {
+      log.error("Error appending WAL file: {}, {}", logFile, e);
+      close();
       throw new ConnectException(e);
     }
   }
 
+  public long getSleepIntervalMs() {
```

Comment:
nit: Could have "package private" scope instead and a non-javadoc comment above to show it's destined for testing.